### ISSUE TITLE
Humble Build Error

### ISSUE THREAD

**Issue by NarenTheNumpkin** [open-navigation/opennav_coverage#77]
I am using Ubuntu 22.04 and ROS2 Humble.

I followed the solution given in the [#58](https://github.com/open-navigation/opennav_coverage/issues/58) but when I try to build the package I'm getting the following error in the screenshot. Also I was using F2C v1.2.1

![Screenshot from 2024-09-28 16-33-19](https://github.com/user-attachments/assets/4171a0b8-4a80-46bf-9c29-1e253508fe06)

**Comment by Ragnar7982** [comment]
@Wxua2005 hello there, did you fix it? If you already fixed it please let me know, thank you.

**Comment by Wxua2005** [comment]
Yes I did but I recommend you not to use this, it's very confusing and lacks documentation and sometimes randomly throws errors during built

**Comment by Ragnar7982** [comment]
@Wxua2005 So do you recommend others method to test complete coverage?
